packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -37,6 +38,8 @@ ThemeService.get().onThemeChange(event => changeTheme(event.newTheme.editorTheme
 @injectable()
 export class MonacoEditorProvider {
 
+    protected diffNavigatorFactory = new MonacoDiffNavigatorFactory();
```

Comment:
please use injection when possible instead of `new` operator

Reply:
done